Makefile.armv7m

```diff
@@ -40,26 +40,27 @@ endif
 ifeq ($(TARGET_SUBFAMILY), stm32l152xd)
 	RAM_SIZE=48
 	VADDR_KERNEL_INIT=8000000
-	KERNEL_TARGET_DEFINE=-DCPU_STM32L1 -DCPU_STM32
+	KERNEL_TARGET_DEFINE=-DCPU_STM32L152XD 
 else ifeq ($(TARGET_SUBFAMILY), stm32l152xe)
 	RAM_SIZE=80
 	VADDR_KERNEL_INIT=8000000
-	KERNEL_TARGET_DEFINE=-DCPU_STM32L1 -DCPU_STM32
+	KERNEL_TARGET_DEFINE=-DCPU_STM32L152XE 
 else ifeq ($(TARGET_SUBFAMILY), stm32l4x6)
 	RAM_SIZE=320
 	VADDR_KERNEL_INIT=8000000
-	KERNEL_TARGET_DEFINE=-DCPU_STM32L4 -DCPU_STM32
+	KERNEL_TARGET_DEFINE=-DCPU_STM32L4X6 
 else ifeq ($(TARGET_SUBFAMILY), imxrt105x)
 	RAM_SIZE=128
 	VADDR_KERNEL_INIT=0
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT105X -DNOMMU 
 else ifeq ($(TARGET_SUBFAMILY), imxrt106x)
 	RAM_SIZE=256
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT106X -DCPU_IMXRT
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT106X -DNOMMU 
 else ifeq ($(TARGET_SUBFAMILY), imxrt117x)
 	RAM_SIZE=256
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT105X -DCPU_IMXRT
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT117X -DNOMMU 
```

Comment:
Why is -DNOMMU needed for imxrt but not for stm32?

Reply:
Probably this flag for stm32 was overlooked while Makefile was creating. I will fix it.